_patterns/error-summary/basic.html

```diff
@@ -0,0 +1,11 @@
+---
+layout: pattern
+---
+
+<div class="error-summary">
+	<div class="error-summary__text">
+		<h4>Submission failed</h4>
+		<p>Check your answers before trying again</p>
+		<p>Label of the field: error message from the system</p>
```

Comment:
Have we considered displaying these in a `<ul>`, as there is potentially a list of them?

Reply:
I now made all the changes here, they were all valid points. And you are correct, `ul` is probably the right way to go. However, I don't like the bullet, I guess that could be removed then in the aplus sass-files with a separate small pr?

Reply:
Yes. You can change the styles to remove the list look.

Reply:
PR for this has now been created: https://github.com/apluslms/a-plus/pull/658